app.js

```diff
@@ -161,8 +162,8 @@ connection.on("open", function () {
   app.get('/zones/:zone/:attribute', function(req, res) {
     zones[req.zone] = undefined;
     connection.write("?10\r");
-    connection.write("?20\r");
-    connection.write("?30\r");
+    AmpCount >= 2 && connection.write("?20\r");
+    AmpCount >= 3 && connection.write("?30\r");
```

Comment:
Is it possible to have 4 amps in a chain, or is 3 the max?

Reply:
3 is the max @jnewland